modules/math/src/geospatial/helpers/scale-to-geodetic-surface.js

```diff
@@ -22,15 +22,15 @@ const scaleToGeodeticSurfaceGradient = new Vector3();
  *
  * @private
  */
-export default function scaleToGeodeticSurface(cartesian, ellipsoid, result) {
+export default function scaleToGeodeticSurface(cartesian, ellipsoid, result = [0, 0, 0]) {
   const {oneOverRadii, oneOverRadiiSquared, centerToleranceSquared} = ellipsoid;
-  const positionX = cartesian.x;
-  const positionY = cartesian.y;
-  const positionZ = cartesian.z;
+  const positionX = cartesian.x || 0;
+  const positionY = cartesian.y || 0;
+  const positionZ = cartesian.z || 0;
 
-  const oneOverRadiiX = oneOverRadii.x;
-  const oneOverRadiiY = oneOverRadii.y;
-  const oneOverRadiiZ = oneOverRadii.z;
+  const oneOverRadiiX = oneOverRadii.x || 0;
```

Comment:
Do we really need these defaults? zero values here seem like an incorrect ellipse, maybe we should `assert` instead?